packages/governance/test/swingsetTests/committeeBinary/vat-voter.js

```diff
@@ -43,7 +43,7 @@ const build = async (log, zoe) => {
         },
       });
       const subscription = E(electoratePublicFacet).getQuestionSubscription();
-      observeIteration(subscription, votingObserver);
+      void observeIteration(subscription, votingObserver);
```

Comment:
I have rarely see the `void` unary operator used. I'm surprised to see it.

Do your linting rules understand the `void` unary operator as allowing the operand to be a floating promise without complaint?

Reply:
> Do your linting rules understand the void unary operator as allowing the operand to be a floating promise without complaint?

Exactly. It's the linter's convention "to explicitly mark a promise as intentionally not awaited."